packages/client/utils/reactjiPickerUtils.ts

```diff
@@ -0,0 +1,28 @@
+const DEFAULT = {
+  heart: 9,
+  tada: 8,
+  smile: 7,
+  rocket: 6,
+  fire: 5,
+  white_check_mark: 4,
+  confused: 3,
+  cry: 2,
+  x: 1
+}
+
+const getRecent = () => {
+  const localStorageRecent = localStorage.getItem('recent')
```

Comment:
I imagine ‘recent’ could be more specific, like ‘recentReactjiPicker’

Reply:
Or ‘frequentlyUsedReactji’

Reply:
I agree. As an improvement,I can create a constant variable, as I am using it in two different places.